src/cascadia/TerminalApp/CascadiaSettingsSerialization.cpp

```diff
@@ -669,9 +670,9 @@ void CascadiaSettings::_LayerOrCreateColorScheme(const Json::Value& schemeJson)
 // Return Value:
 // - a ColorScheme that can be layered with the given json object, iff such a
 //   color scheme exists.
-ColorScheme* CascadiaSettings::_FindMatchingColorScheme(const Json::Value& schemeJson)
+implementation::ColorScheme* CascadiaSettings::_FindMatchingColorScheme(const Json::Value& schemeJson)

```

Comment:
oh hey this should probably not be a bare `implementation::ColorScheme*` anymore. It should probably be the `com_ptr<implementation::ColorScheme>` now, right?